### PR TITLE
[AUTO-MERGE] Merge main into develop via merge-main-to-develop

### PR DESCRIPTION
**Pull request by idanov**
A new change in main cannot be merged into develop as part of the regular sync job, hence this PR. Please resolve the conflicts manually, and make sure to obtain 2 approvals once the builds pass.

### IMPORTANT NOTICE

Please let CircleCI merge this PR automatically, with merge commit enabled.

### CODE REVIEW

**Review by lorenabalan — APPROVED** [review]
Nicely done, thank you for wrestling the conflicts!

File: setup.py [hunk 1]
```diff
@@ -83,7 +83,12 @@ def _collect_requirements(requires):
     "pandas.XMLDataSet": [PANDAS, "lxml~=4.6"],
     "pandas.GenericDataSet": [PANDAS],
 }
-pillow_require = {"pillow.ImageDataSet": ["Pillow~=8.0"]}
+pillow_require = {
+    "pillow.ImageDataSet": [
+        "Pillow~=9.0; python_version > '3.6'",
+        "Pillow~=8.0; python_version == '3.6'",
+    ]
+}
```
Comment (AntonyMilneQB) [inline setup.py:91]:
```suggestion
pillow_require = {"pillow.ImageDataSet": ["Pillow~=9.0"]}
```

File: test_requirements.txt [hunk 2]
```diff
@@ -28,7 +28,7 @@ networkx~=2.4
 openpyxl>=3.0.3, <4.0
 pandas-gbq>=0.12.0, <1.0
 pandas~=1.3  # 1.3 for read_xml/to_xml
-Pillow~=8.0
+Pillow~=9.0; python_version > '3.6'
```
Comment (AntonyMilneQB) [inline test_requirements.txt:31]:
```suggestion
Pillow~=9.0
```

**Review by AntonyMilneQB — APPROVED** [review]
Thanks for doing this! Just a couple of small simplifications since develop doesn't need to support Python 3.6.

**Pull request merged**